### ISSUE TITLE
Magento 2 Fresh Install - Admin can't add/view products, categories, pages, etc.

### ISSUE THREAD

**Issue by macguydvd** [magento/magento2#12897]
### Preconditions

1. Fresh Ubuntu 16.04 LTS install
2. PHP 7.0.22
3. MySQL 5.7.20
4. Apache 2.4.18
5. Fresh Magento 2.2.2 Install

### Steps to reproduce

1. Install Magento (I have tried via browser and composer, with or without sample data)
2. Launch and login to admin
3. Navigate to Catalog > Products

### Expected result

1. On another machine I see products at this point, and if I click Add Product I get a form. Same thing with categories, or pages.

### Actual result

1. I see the spinner for a few seconds and then, nothing. There are no apache errors or errors in the javascript console. Also happens whether or not the sample data is loaded. The front end of the store shows all the products and pages as expected.

Here are some screenshots.

![screen-shot-2017-12-27-at-10 51 03-am](https://user-images.githubusercontent.com/26262182/34388995-39030fc8-eaf4-11e7-9790-943d7615855d.jpg)

![screen-shot-2017-12-27-at-10 51 37-am](https://user-images.githubusercontent.com/26262182/34388999-3e04b8dc-eaf4-11e7-9e39-de007abd1d86.jpg)

**Comment by mombrea** [comment]
When in doubt, execute this giant pile of commands and try again. We have to do this non stop during development.

`rm -rf var/di var/generation generated/code`

**Clean and Rebuild**

`php bin/magento cache:clean`
`php bin/magento cache:flush`
`php bin/magento setup:upgrade`
`php bin/magento setup:di:compile`
`php bin/magento indexer:reindex`
`php bin/magento setup:static-content:deploy`

**Reset Permissions**
`chown <user>:<group> -R *`

`find . -type f -exec chmod 644 {} \;`
`find . -type d -exec chmod 755 {} \; `
`find ./var -type d -exec chmod 777 {} \;`
`find ./pub/media -type d -exec chmod 777 {} \;`
`find ./pub/static -type d -exec chmod 777 {} \;`
`chmod 777 ./app/etc`
`chmod 644 ./app/etc/*.xml`

**Comment by macguydvd** [comment]
Thanks for the suggestions, there are a few there I wasn't already using and I'm sure they will come in handy. Of course after several days of trying to figure this out, the answer presented itself as soon as I posted an issue here.

For me the problem was mod_pagespeed, specifically the command:
`ModPagespeedEnableFilters pedantic,remove_comments`
This was inserted based upon a tutorial online and as soon as I commented it out, everything came to life on the admin side of Magento.

Thanks again for the great list of Magento tools!
